src/System.Net.Mail/src/System/Net/Mail/SmtpConnection.cs

```diff
@@ -172,7 +167,13 @@ internal void Abort()
                         // interpreted correctly by the server if it's in the middle of a
                         // DATA command or some similar situation.  This may send a RST
                         // but this is ok in this situation.  Do not reuse this connection
-                        _tcpClient.LingerState = new LingerOption(true, 0);
+                        try
+                        {
+                            _tcpClient.LingerState = new LingerOption(true, 0);
+                        }
+                        catch
```

Comment:
Can we catch the exact problematic exception type?

Reply:
The dump indicates that we don't get past this line.
But we don't know what exact exception gets thrown.